src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -3513,38 +3573,10 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
                 }
                 break;
             }
-            case (ASR::ttypeType::UnsignedInteger) : {
```

Comment:
This needs to be reverted: https://github.com/lcompilers/lpython/pull/1764/files#diff-48a7161397f9eb8b730f0cbec2f11ce1471aac961dc6b16dc558cf161c375054